types/index.d.ts

```diff
@@ -55,6 +55,7 @@ declare namespace Undici {
   const MockClient: typeof import('./mock-client').default
   const MockPool: typeof import('./mock-pool').default
   const MockAgent: typeof import('./mock-agent').default
+  const MockCallHistory: typeof import('./mock-call-history')
```

Comment:
Don't know if this is the right with to export types. `mock-call-history.d.ts` is exporting with named export

Reply:
Found my answer